src/reanimated2/globals.d.ts

```diff
@@ -111,4 +112,5 @@ declare global {
   var UpdatePropsManager: UpdatePropsManager;
   var ProgressTransitionRegister: ProgressTransitionRegister;
   var updateJSProps: (viewTag: number, props: Record<string, unknown>) => void;
+  var RNScreensTurboModule: RNScreensTurboModuleType;
```

Comment:
We cannot assume that the module exists, let's do this for better type safety.
```suggestion
  var RNScreensTurboModule: RNScreensTurboModuleType | undefined;
```